### PR TITLE
[CPM-96] Refactor Frontend Structure

### PR DESCRIPTION
**Pull request by FocalChord**

### CODE REVIEW

File: client/src/__tests__/Pages/CoursesPage/CoursesPage.test.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
 import React from "react";
 import renderer from "react-test-renderer";
 import { AppConfiguration } from "../../../components";
-import { CoursesPage } from "../../../pages";
+import { Courses } from "../../../pages";
 
 test("<CoursesPage />", () => {
```
Comment (vee-christella) [inline client/src/__tests__/Pages/CoursesPage/CoursesPage.test.js:6]: Can we rename this to `Courses` now?

**Review by vee-christella — APPROVED** [review]
When should we refactor common components? 🤔 The rest looks CLEAN though! 💧

File: client/src/__tests__/Pages/CoursesPage/CoursesPage.test.js (hunk 1 above)
Comment (FocalChord) [reply client/src/__tests__/Pages/CoursesPage/CoursesPage.test.js:6]: I'll make a ticket for refactoring common components as this is for structure :D

**Review by Dinith1 — APPROVED** [review]
Noice, LGTM :shipit:

**Pull request merged**